src/app/user/settings/page.tsx

```diff
@@ -1,231 +1,337 @@
+                      : "border-slate-300 text-slate-700 hover:bg-gray-200 dark:text-slate-300 dark:hover:bg-slate-800"
+                  }`}
+              >
+                {t === "email"
+                  ? "Update Email"
+                  : t === "password"
+                  ? "Update Password"
+                  : "Delete Account"}
+              </button>
+            ))}
+          </div>
 
-        <form noValidate className="mb-2">
-          <FormField
-            id="currentEmail"
-            label="Email"
-            type="email"
-            value={currentEmail}
-            onChange={() => {}}
-          />
-          <FormField
-            id="currentPassword"
-            label="Password"
-            type="password"
-            value={currentPassword}
-            onChange={(e) => setCurrentPassword(e.target.value)}
-          />
-        </form>
-
-        {!!error ? (
-          <Alert variant="destructive">
-            <AlertTitle>{error}</AlertTitle>
-          </Alert>
-        ) : null}
-        {!!success ? (
-          <Alert>
-            <AlertTitle>{success}</AlertTitle>
-          </Alert>
-        ) : null}
-
-        {tab === "email" && (
-          <form noValidate onSubmit={handleEmailUpdate}>
-            <FormField
-              id="newEmail"
-              label="New Email"
-              type="email"
-              value={newEmail}
-              onChange={(e) => setNewEmail(e.target.value)}
-            />
-
-            <Button
-              type="submit"
-              className="w-full my-4 bg-gray-400 hover:bg-gray-500 dark:bg-slate-700 dark:hover:bg-slate-800 text-base text-black dark:text-white"
-            >
-              Save Email
-            </Button>
-          </form>
-        )}
-
-        {tab === "password" && (
-          <form noValidate onSubmit={handlePasswordUpdate}>
-            <FormField
-              id="newPassword"
-              label="New Password"
-              type="password"
-              value={newPassword}
-              onChange={(e) => setNewPassword(e.target.value)}
-            />
-
-            <Button
-              type="submit"
-              className="w-full my-4 bg-gray-400 hover:bg-gray-500 dark:bg-slate-700 dark:hover:bg-slate-800 text-base text-black dark:text-white"
-            >
-              Save Password
-            </Button>
-          </form>
-        )}
-
-        {tab === "delete" && (
-          <div>
-            <p className="text-sm text-slate-700 dark:text-slate-300 text-center mb-3">
-              Permanently delete your account.
-            </p>
-            <Button
-              type="button"
-              variant="destructive"
-              onClick={handleDelete}
-              className="w-full"
-            >
-              Delete My Account
-            </Button>
+          {!!error && (
+            <Alert variant="destructive" className="mb-3 text-red-500">
```

Comment:
The success alert incorrectly uses red text color (text-red-500), which is typically associated with errors. This creates a confusing user experience. Consider using a success-appropriate color.